fe/fe-core/src/main/java/com/starrocks/analysis/DropColumnClause.java

```diff
@@ -67,7 +58,7 @@ public String toSql() {
         StringBuilder sb = new StringBuilder();
         sb.append("DROP COLUMN `").append(colName).append("`");
         if (rollupName != null) {
-            sb.append(" IN `").append(rollupName).append("`");
+            sb.append(" FROM `").append(rollupName).append("`");
```

Comment:
Why  need to change the `In` to `From`?  we should the behavior is consistent.

Reply:
Revert these change.